src/dto/mats-monitor-hrly-value.dto.ts

```diff
@@ -0,0 +1,27 @@
+export class MatsMonitorHrlyValueBaseDTO {
+  parameterCode: string;
+  unadjustedHourlyValue?: string;
+  modcCode?: string;
+  monitoringSystemId?: string;
```

Comment:
I think the id for componentId and monitoringSystemId should be capitalized so "componentID" and "monitoringSystemID". Reason being is that that's how the json schema  was defined to make it closer with the xml.